src/aparnaVDec19/assignment2/SecondLargestNumberInArray.java

```diff
@@ -0,0 +1,28 @@
+package aparnaVDec19.assignment2;
+
+/*
+ * 5)Write a java program to find second largest element in an array of integers?
+ */
+public class SecondLargestNumberInArray {
+
+	static void seconLargestNumberFind(int[] arr) {
+		int largest = arr[0];
+		int secondLargest = arr[0];
+		for (int i = 1; i < arr.length; i++) {
+			if (arr[i] > largest) {
+				secondLargest = largest;
+				largest = arr[i];
+
+			} else if (arr[i] > secondLargest) {
```

Comment:
here second is like that arr[i] should not be largest number otherwise you get secondLargest number and largest number same... hope you get idea
 
currentNumber is greater than secondLargetrs **and** currentNumber should not be largest number